openapi/adintel/adintel.yaml

```
@@ -192,6 +287,51 @@ components:
         status: trustedTester
       x-tags:
         - accountStats
+    connectedAccount:
+      title: connectedAccount
+      type: object
+      description: 'An advertising account that’s connected within Advertising Intelligence from integrated campaign management platforms, including Google Adwords, Facebook Ads and Local Ads.'
+      x-examples: {}
+      properties:
+        id:
+          type: string
+          description: The resource ID of the Advertising Account
+        type:
+          type: string
+          description: connectedAccount
+        attributes:
+          type:
+            - string
```

```suggestion
          type: object
```

Should be of type object